settings.conf

```diff
@@ -51,15 +51,13 @@ sequence:
   - locale
   - keyboard
   - localecfg
-  - luksbootkeyfile
   - users
   - displaymanager
   - networkcfg
   - hwclock
   - contextualprocess@before_bootloader_mkdirs
   - shellprocess@bug-LP#1829805
   - initramfscfg
-  - initramfs
```

Comment:
Seems important so I will be doing some checking on it, but I couldn't find it in the source tree, and the logs complain that it doesn't exist.